examples/EncodingWithS3InputAndS3OutputMP4WithStreamConditionsForAudioDropStream.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+use Bitmovin\api\ApiClient;
+use Bitmovin\api\enum\AclPermission;
+use Bitmovin\api\enum\CloudRegion;
+use Bitmovin\api\enum\codecConfigurations\H264Profile;
+use Bitmovin\api\enum\encodings\ConditionAttribute;
+use Bitmovin\api\enum\encodings\ConditionOperator;
+use Bitmovin\api\enum\SelectionMode;
+use Bitmovin\api\enum\Status;
+use Bitmovin\api\exceptions\BitmovinException;
+use Bitmovin\api\model\codecConfigurations\AACAudioCodecConfiguration;
+use Bitmovin\api\model\codecConfigurations\H264VideoCodecConfiguration;
+use Bitmovin\api\model\encodings\Encoding;
+use Bitmovin\api\model\encodings\helper\Acl;
+use Bitmovin\api\model\encodings\helper\EncodingOutput;
+use Bitmovin\api\model\encodings\helper\InputStream;
+use Bitmovin\api\model\encodings\muxing\MP4Muxing;
+use Bitmovin\api\model\encodings\muxing\MuxingStream;
+use Bitmovin\api\model\encodings\streams\condition\Condition;
+use Bitmovin\api\model\encodings\streams\Stream;
+use Bitmovin\api\model\inputs\S3Input;
+use Bitmovin\api\model\outputs\S3Output;
+use Bitmovin\api\enum\muxing\StreamConditionsMode;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$bitmovinApiKey = 'YOUR_BITMOVIN_API_KEY';
+$cloudRegion = CloudRegion::AUTO;
+
+// S3 INPUT CONFIGURATION
+$s3InputAccessKey = 'YOUR_ACCESS_KEY';
+$s3InputSecretKey = 'YOUR_SECRET_KEY';
+$s3InputBucketName = 'YOUR_BUCKETNAME';
+$videoInputPath = 'path/to/your/inputfile.mp4';
+
+$s3OutputAccessKey = 'YOUR_ACCESS_KEY';
+$s3OutputSecretKey = 'YOUR_SECRET_KEY';
+$s3OutputBucketName = 'YOUR_BUCKETNAME';
+$outputPath = 'path/to/your/output/destination/';
+
+$videoEncodingProfiles = array(
+    array("height" => 1080, "bitrate" => 4800000, "profile" => H264Profile::HIGH),
```

Comment:
Please use the bitrate ladder shown in the example I linked earlier as it is the "recommended general purpose bitrate ladder" for encoding. Thank you

Reply:
https://github.com/bitmovin/bitmovin-php/blob/develop/examples/EncodingWithS3InputAndS3Output.php#L55